### PR TITLE
Prepared for production

### PR DESCRIPTION
**Pull request by Vovik1**

### CODE REVIEW

File: emails/signup.js [hunk 1]
```diff
@@ -1,19 +1,15 @@
-const { EMAIL_FROM, BASE_URL } = require('../config/keys');
-
-const signupEmail = (email) => (
-  {
-    from: EMAIL_FROM,
-    to: email,
-    subject: 'Регістрація нового користувача пройшла успішно',
-    html: `
+const signupEmail = (email) => ({
```
Comment (Antoniozinchenko) [inline emails/signup.js:1]: Missing import BASE_URL

File: config/keys.js [hunk 2]
```diff
@@ -1,27 +1,3 @@
-const SECRET_JWT_KEY_AUTH = 'defibrillator_secret';
-const EXPIRE_TIME_JWT_AUTH = 60 * 60; //seconds
-const SECRET_JWT_KEY_SIGN_UP = 'signup_create_secret';
-const EXPIRE_TIME_JWT_SIGN_UP = 60 * 60; //seconds
-const SECRET_JWT_KEY_RESET = 'reset_create_secret';
-const EXPIRE_TIME_JWT_RESET = 60 * 60; //seconds
-const EMAIL_USER = 'defibrillator.lviv@gmail.com';
-const EMAIL_PASSWORD = 'Defibrillator1!';
-const EMAIL_FROM = 'defibrillator.lviv@gmail.com';
 const BASE_URL = 'http://localhost:3000';
```
Comment (Antoniozinchenko) [inline config/keys.js:1]: BASE_URL should be also in env config

File: client/src/shared/api/index.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import http from '../http';
 
-const host = window.location.origin;
+const host = 'http://localhost:3000';
```
Comment (Antoniozinchenko) [inline client/src/shared/api/index.js:3]: revert it

**Comment by Vovik1** [comment]
Fix BaseUrl

**Pull request closed without merging**